backend/app/services/template_seeds.py

```
@@ -0,0 +1,256 @@
+    created_count = 0
+
+    for tpl_data in BUILT_IN_TEMPLATES:
+        name = tpl_data["name"]
+
+        existing = (
+            db.query(Kind)
+            .filter(
+                Kind.kind == TEMPLATE_KIND,
+                Kind.name == name,
+                Kind.namespace == TEMPLATE_NAMESPACE,
+                Kind.is_active == True,
```

Ruff (0.15.10)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)